.kotlin_alpha/services/textract/src/main/kotlin/com/kotlin/textract/DetectDocumentTextS3.kt

```diff
@@ -0,0 +1,89 @@
+// snippet-sourcedescription:[DetectDocumentText.kt demonstrates how to detect text in the input document.]

```

Comment:
DetectDocumentText**S3**.kt
Also, maybe update comment to reflect S3-specific?

Reply:
fixed